### DISCUSSION TITLE
dll files could not be found

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#9085] [Q&A] [answered]
Hi,

This weekend i upgraded a solution to the new version of Orchard Core. Since then I'm not able to build my solution.
I've already updated my Visual Studio but now I'm kinda stuck. 

![image](https://user-images.githubusercontent.com/61787638/113831946-6de4f500-9788-11eb-90fa-72b4e3db9d66.png)

Does someone have any idea to fix this?

**Comment by EricBui619** [comment]
Let's try close solution, delete bin and obj folder and then restart you project again

**Reply by craftyweb** [reply]
I've tried this several times but without success

**Comment by deanmarcussen** [comment, marked as answer]
It looks like your custom themes are the ones not being found.

We upgrade dev recently to multi target net5 and core 3.

You might want to take a look at the pr, and associated changes that happen with .csproj files and see if you are missing something in those files. https://github.com/OrchardCMS/OrchardCore/pull/8550

**Reply by craftyweb** [reply]
Thanks! I'll check it out

**Reply by craftyweb** [reply]
Thank you!

When creating the themes and modules with the code generation templates the TargetFramework was netcore3.1 so I replaced this with $(AspNetCoreTargetFramework) and it works now
